src/index.js

```diff
@@ -29,8 +29,9 @@
 /* eslint-disable no-unused-vars */
 
 'use strict';
-
+import 'babel-polyfill';
```

Comment:
Remove this line. It should not be needed.

Reply:
removed